ui/docs-app/docs_app/theme/assets/semantic-ui/js/docs_app/search/components/ResultsItemAccessStatus.jsx

```diff
@@ -8,7 +8,7 @@ export const ResultsItemAccessStatus = ({ status }) => {
     iconFile && (
       <Image
         as="a"
-        href={`/vocabularies/${type}/${id}`}
```

Comment:
Why not use type from props? is it empty & why? If it couldn't be passed via type prop, remove the prop as it will be unused

Reply:
@mirekys Zdar, the vocabulary info in metadata looks like this

![image](https://github.com/Narodni-repozitar/nr-docs/assets/34883098/6e63101a-3d43-4dcc-8713-f005f696ba51)

So only the basic data for showing it and no type included. As the component is related specifically to access-rights vocabulary, I thought hardcoding it would not be such a problem. I removed the type from props.